post-notifs.py

```diff
@@ -95,7 +95,7 @@ def spam_notifications(notif_type, formatter_class, talk_page, files):
     try:
         talk_page.save(summary=summary, botflag=wiki_options['markasbot'], minor=wiki_options['minoredit'], **kvargs)
         print('Posted a notification about %d %s files to %s' % (len(ourlist), notif_type, talk_page))
-    except pywikibot.LockedPage:
+    except pywikibot.exceptions.LockedPageError:
         print('Page %s is protected, skipping' % talk_page, file=sys.stderr)
```

Comment:
This will fix the `AttributeError`. I suggest updating this to something like the below to catch all page save-related exceptions.

```python
    except pywikibot.exceptions.PageSaveRelatedError as e:
        print(e + ' Skipping.', file=sys.stderr)
```

Reply:
Brilliant! `PageSaveRelatedError` seems very nifty. I'm really starting to like this framework :)

I'd like to also log the page title here in case we debug an issue, so I went with `'Unable to edit page %s due to: %s' % (talk_page, str(e))` which I think should work?

Reply:
That will work. For `PageSaveRelatedError`, `str(e)` will already contain the page title though.
`PageSaveRelatedError` catches `LockedPageError`, so you could just have one `except` branch.

Reply:
Perfecto! Done and thanks.